src/lib.rs

```diff
@@ -68,11 +68,13 @@ where
         v.push(5u64);
         tracing::info!(vec = ?v, vec.addr = ?v.as_ptr());
         v.push(10u64);
-        tracing::info!(vec=?v, vec.addr = ?v.as_ptr());
+        tracing::info!(vec=?v, vec.addr=?v.as_ptr());
         assert_eq!(v.pop(), Some(10));
         assert_eq!(v.pop(), Some(5));
     }
 
+    panic!("fake panic!");
```

Comment:
We prob. want to remove this before we merge

Reply:
oopsie-whoopsie!